src/rest/run_method.ts

```diff
@@ -62,7 +62,11 @@ export async function runMethod<T = any>(
         method,
         reject,
         respond: (data: { status: number; body?: string }) =>
-          resolve(camelize<T>(JSON.parse(data.body || "{}"))),
+          resolve(
+            data.status !== 204
+              ? camelize<T>(JSON.parse(data.body ?? "{}"))
+              : <T> (<unknown> undefined),
```

Comment:
```suggestion
              : (undefined as unknown) as T,
```
Both of these are equivalent to each other, however, using `as` for type assertions is the recommended syntax.